### ISSUE TITLE
Atmosphere doesn't boot after updating.

### ISSUE THREAD

**Issue by Troy2424** [HamletDuFromage/aio-switch-updater#90]
I'm trying to update atmosphere using AIO Switch Updater. But the problem is AIO updater says it will boot to a special payload once it's done downloading atmosphere. What payload is this? because once the console reboots to finish installing the update, I get this screen.
![img1 daumcdn](https://user-images.githubusercontent.com/57131083/117027062-52f39b00-acca-11eb-8348-4258fc8fcf00.jpg)
And then I press any button to continue and it just says something like atmosphere encountered fatal error. and reboots again in a loop. Do I HAVE TOO use Hekate with AIO updater, because it says update your bootloader folder but I don't use hekate and therefore don't use a bootloader folder. I just use vanilla atmosphere with fusee primary.bin. I never had this problem before in older versions of AIO updater. The last time I used AIO updater was version 2.3.4 and I didn't have this problem.

**Comment by HamletDuFromage** [comment]
You have downloaded Hekate at some point, and now you have an (outdated) Hekate payload named /bootloader/update.bin.
Delete that, inject fusee-primary and you'll be good to go

**Comment by Troy2424** [comment]
I have never downloaded hekate and I don't have a /bootloader/update.bin folder to delete. Just to test, I deleted everything on my SD card and installed a fresh older version of atmosphere. And tried using AIO to update it and I still ran into the same problem.

**Comment by HamletDuFromage** [comment]
uh, that's weird then. I'll take a look at it.
It's supposed to inject update.bin if it exists, and reboot_payload otherwise. Maybe your reboot_payload is secretly hekate?
https://github.com/HamletDuFromage/aiosu-rcm/blob/e08b6b7ee63a6238716aad3c5dc96860d1bf9c2c/bootloader/main.c#L439

**Comment by HamletDuFromage** [comment]
Actually, if it's injecting Hekate, it means you have an Hekate somewhere on your sd card

**Comment by Troy2424** [comment]
That's impossible, I don't know how hekate could be on my SD card. I deleted everything on my sd card and downloaded a fresh version of atmosphere. and reboot_payload isn't secretly hekate since atmospheres default isn't hekate and I just downloaded atmosphere. Here is my SD card with atmosphere version 0.18.1. if you wanna take a look. https://drive.google.com/drive/folders/1wYhNLFR9ebukn-cX7E83-y_HeAyIbQP-?usp=sharing
